### PR TITLE
Add API to get current user from server

### PR DESCRIPTION
**Pull request by ben181231**
connect SkygearIO/skygear-server#111

### CODE REVIEW

File: lib/container.js [hunk 1]
```diff
@@ -181,6 +181,16 @@ export default class Container {
...
   }
 
+  whoami() {
+    let container = this;
+    return container.makeRequest('me', {})
+    .then(function (res) {
+      return new container.User(res.result);
+    }, function (err) {
+      return Promise.reject(err);
+    });
```
Comment (rickmak) [inline lib/container.js]: I think `whoami` will fetch the remote user AND update the `_user` (i.e. keep the `_currentUser` in sync with remote.

Comment (ben181231) [inline lib/container.js]: Do you expect the `me` request will return the access token ? 

Currently, the `access_token` field is a required field in the view of SDK, but the `me` endpoint will not return it. Should I inject the token back the returned object and set it to `_currentUser` ?

Comment (rickmak) [inline lib/container.js]: Yes, let do this. It can also act as a access_token rotation in future.

**Comment by ben181231** [comment]
@rickmak The PR updated.

**Pull request merged**